static/app/stories/view/storyTableOfContents.tsx

```diff
@@ -200,14 +213,25 @@ function StoryContentsList({
   const LinkComponent = isChild ? StyledChildLink : StyledLink;
 
   return (
-    <li>
+    <Flex as="li" direction="column">
```

Comment:
We should probably handle this on the `Container` level, but changing the `display` of `li` usually requires `role="listitem"` to reset the semantics for screen readers.